tests/test_bias.py

```diff
@@ -0,0 +1,168 @@
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import os
+import numpy as np
+import unittest
+
+import lsst.afw.math as afwMath
+import lsst.daf.persistence as dafPersist
+import lsst.ip.isr as ipIsr
+import lsst.meas.algorithms as measAlg
+import lsst.utils.tests
+
+from lsst.pipe.tasks.repair import RepairTask
+
+
+# TODO: DM-26396
+#       Update these tests to validate calibration construction.
+class BiasTestCases(lsst.utils.tests.TestCase):
+
+    def setUpClass(self):
+        """Setup butler, and generate an ISR processed exposure.
+
+        Notes
+        -----
+        DMTN-101 4.1:
+
+        Process an independent bias frame through the ISR including
+        overscan correction and bias subtraction
+
+        """
+        repoDir = os.path.join("..", "DATA")
+        calibDir = os.path.join("..", "DATA", "calibs")
+        butler = dafPersist.Butler(repoDir, calibRoot=calibDir)
+
+        self.config = ipIsr.IsrTaskConfig()
+        self.config.doSaturation = True
+        self.config.doSuspect = True
+        self.config.doSetBadRegions = True
+        self.config.doOverscan = True
+        self.config.doBias = True
+        self.config.doVariance = True
+
+        self.config.doLinearize = False
+        self.config.doCrosstalk = False
+        self.config.doWidenSaturationTrails = False
+        self.config.doBrighterFatter = False
+        self.config.doDefect = False
+        self.config.doSaturationInterpolation = False
+        self.config.doDark = False
+        self.config.doStrayLight = False
+        self.config.doFlat = False
+        self.config.doApplyGains = False
+        self.config.doFringe = False
+        self.config.doMeasureBackground = False
+        self.config.doVignette = False
+        self.config.doAttachTransmissionCurve = False
+        self.config.doUseOpticsTransmission = False
+        self.config.doUseFilterTransmission = False
+        self.config.doUseSensorTransmission = False
+        self.config.doUseAtmosphereTransmission = False
+
+        self.isrTask = ipIsr.IsrTask(config=self.config)
+        # This is not an independent frame.
```

Comment:
This might warrant a proper `TODO:` and a ticket number, given that having some reserved biases etc means a new section of the manifest I assume, and retrieving that so that the line below isn't hard coding the visit.